obelisk_ws/src/obelisk_ros/config/dummy_cpp.yaml

```diff
@@ -82,6 +82,21 @@ onboard:
           sensor_settings:
           - topic: /obelisk/dummy/sensor
             dt: 0.001
-            sensor_type: jointpos
+            msg_type: ObkJointEncoders
             sensor_names:
-            - sensor_joint1
+            - joint_pos: jointpos
+              joint_vel: jointvel
```

Comment:
make the value for these `joint_pos` and `joint_vel` to match the field name in the message type. this means you also need to change the parsing code to look for the underscore version in the mujoco sim robot.

Reply:
also do this for the dummy_cpp_viz robot

Reply:
i resolved this